### PR TITLE
Update KML Viewer demo for v200

### PR DESCRIPTION
**Pull request by nCastle1**

### CODE REVIEW

File: src/KmlViewer/XInputHelper/XInputSceneController.cs [hunk 1]
```diff
@@ -166,8 +166,7 @@ private void UpdateView()
             if (state.Gamepad.Buttons.HasFlag(GamepadButtonFlags.DPadRight))
                 newCamera = newCamera.RotateTo(90, newCamera.Pitch, newCamera.Roll);
 
-            if (!newCamera.IsEqual(originalCamera))
-                sceneView.SetViewpointCamera(newCamera);
```
Comment (dotMorten) [inline src/KmlViewer/XInputHelper/XInputSceneController.cs:170]: I believe this call was pretty important. While IsEqual was removed, we should likely just do a value comparison instead.

Comment (nCastle1) [reply src/KmlViewer/XInputHelper/XInputSceneController.cs]: I added an implementation, but with my current setup I don't think I'll be able to test it.

File: src/KmlViewer/XInputHelper/XInputSceneController.cs [hunk 2]
```diff
@@ -203,5 +205,45 @@ private static void OnControllerPropertyChanged(DependencyObject d, DependencyPr
...
         }
+
+        private static bool CamerasAreEqual(Camera lhs, Camera rhs)
+        {
+            if (ReferenceEquals(lhs, rhs))
+            {
+                return true;
+            }
+
+            if (lhs == null && rhs == null)
```
Comment (mstefarov) [inline src/KmlViewer/XInputHelper/XInputSceneController.cs:216]: `ReferenceEquals` already checks whether _both_ are null.
Instead, this should check whether _one_ is null.  Checking here will avoid potential NullReferenceExceptions lower in the method.
```suggestion
            if (lhs == null || rhs == null)
```

**Review by mstefarov — COMMENTED** [review]
The app is unusable in dark mode:

2022-11-20_234630 ApplicationFrameHost

If there's no time to add dark mode support now, you can add this to **App.xaml** to always run in light mode:
```xml
<Application
    ...snip...
    RequestedTheme="Light">
```

**Pull request merged**